packages/governance/src/votes/interface.cairo

```diff
@@ -35,7 +35,6 @@ pub trait IVotes<TState> {
 }
 
 /// Common interface for tokens used for voting(e.g. `ERC721Votes` or `ERC20Votes`)
```

Comment:
You mean removing the trait or removing the comment or making it just a code comment and not `///`? I think having the trait is valuable since it forces contracts that use the `VotesComponent` to have an implementation of this trait and it allows someone that wants to use their own token voting mechanism to implement it